packages/apollos-api/src/data/campus/index.js

```diff
@@ -18,6 +20,49 @@ class dataSource extends originalDataSource {
     }
     return [];
   }
+
+  updateCurrentUserCampus = async ({ campusId }) => {
+    const { Auth } = this.context.dataSources;
+
+    const currentUser = await Auth.getCurrentPerson();
+    const { id: rockCampusId } = parseGlobalId(campusId);
+
+    const campus = await this.request()
+      .filter(`Id eq ${rockCampusId}`)
+      .first();
+
+    if (!campus) return null;
+
+    await this.post(
+      `/People/AttributeValue/${currentUser.id}?attributeKey=AppCampus&attributeValue=${campus.guid}`
+    );
+
+    return currentUser;
+  };
+
+  getForPerson = async ({ personId }) => {
+    const APP_CAMPUS_ATTRIBUTE_ID = 132803;
```

Comment:
do you see this as a temporary hack or what you want to do long term

Reply:
It's what I want to do long term for Willow, assuming they don't change their mind on campus ids.